testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/reactive/messaging/rest/channels/ReactiveMessagingChannelsTestCase.java

```diff
@@ -98,16 +92,8 @@ public static WebArchive getDeployment() {
                         // ends up throwing a ClassNotFoundException since it defaults to the GlassFish implementation
                         // which should not be used. Behind the scenes (which gets swallowed, it throws an
                         // AccessControlException due to the missing file.
-                        new FilePermission("<<ALL FILES>>", "read"),
-                        // This needs fixing in RestEasy probably
-                        new SecurityPermission("insertProvider"),
-                        // These can be removed once https://github.com/smallrye/smallrye-reactive-messaging/pull/1334
-                        // has been merged and released.
-                        new MBeanServerPermission("createMBeanServer"),
-                        new MBeanPermission("*", "registerMBean, unregisterMBean"),
-                        new RuntimePermission("getClassLoader"),
-                        new RuntimePermission("modifyThread"),
-                        new RuntimePermission("setContextClassLoader")), "permissions.xml");
+                        new FilePermission("<<ALL FILES>>", "read")
```

Comment:
Is this one still needed? WFLY-15084 should have fixed that at least for Jakarta REST.

Reply:
Good catch - thanks @jamezp